src/tools/tests/bundle-test.ts

```diff
@@ -60,6 +60,21 @@ describe('Bundle Tool', () => {
         ]
     );
   });
+  it('fails for a non existent manifest', done => {
+    bundle(['src/tools/tests/test-data/nope.recipes'], 'test-output/bundle/nope.zip', false)
+        .then(_ => assert.fail('should have failed'))
+        .catch(e => done());
+  });
+  it('fails for a syntax error', done => {
+    bundle(['src/tools/tests/test-data/invalid/syntax-problem.manifest'], 'test-output/bundle/nope.zip', false)
+        .then(_ => assert.fail('should have failed'))
+        .catch(e => done());
+  });
+  it('fails for a non existent file reference', done => {
+    bundle(['src/tools/tests/test-data/invalid/reference-problem.manifest'], 'test-output/bundle/nope.zip', false)
+        .then(_ => assert.fail('should have failed'))
```

Comment:
Can we use `assert.throws` to check that the bundling is throwing a reasonable error?

Reply:
assert.throws deals with synchronous code, while here I'm dealing with async one. I didn't find a way to utilize assert.throws here, let me know if you know how to do it.

I've added asserts to verify that the error thrown described the expected problem, thanks for the suggestion.

Reply:
Thanks :+1: